src/Http/Requests/UpdateFormBuilderResponseRequest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace FluxErp\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdateFormBuilderResponseRequest extends FormRequest
+{
+    public function rules(): array
```

Comment:
should this even be updateable? Like I am quite uncertain about this validation and therefore even about the update route

Reply:
done